android-sdk/src/androidTest/java/com/mobilecoin/lib/TestFogConfig.java

```diff
@@ -113,6 +122,7 @@ private static TestFogConfig getFogConfig(Environment.TestEnvironment testEnviro
                 "fog://fog.%s.mobilecoin.com",
                 testEnvironment.getName()
         ));
+        final Uri shortFogUri = Uri.parse(testEnvironment == Environment.TestEnvironment.TEST_NET ? "fog://fog-rpt-stg.namda.net" : "");
```

Comment:
If this data is needed for only 1 test and it's valid for only 1 environment, adding it to TestFogConfig for all tests/environments creates confusion. Long URIs are used to initialize the clients, and short URI is used in the PublicAddress so clients know where to download the reports from. In this code, the short uri is ignored by all the tests except one. If the short URI is needed for only 1 test it's probably better to isolate the data within that scope.
Another option is to implement it to properly affect all the environments, but I don't see any gains from that at this point.

Reply:
Yeah, I was thinking about that. The reason why I put it here instead of just in the one test was in case we start using short URIs for other environments. But I can just move this to the one test and take it out of the config.